### ISSUE TITLE
Try go1.12

### ISSUE THREAD

**Issue by bsiegert** [bsiegert/BulkTracker#16]
https://groups.google.com/forum/#!topic/google-appengine-go/MFTqqCSl-po

Set `api_version: go1.9` in `app.yaml`.

**Comment by bsiegert** [comment]
I'll leave this running for a bit and see if latency or something improves.

**Comment by bsiegert** [comment]
Got this mail:

> We’re writing to let you know that Go versions prior to 1.11 are no longer supported on App Engine, and new deployments using these language versions (1.6, 1.8, and 1.9) will not be available starting October 1, 2019.

**Comment by bsiegert** [comment]
It turns out that this is a bigger project, since all of the "appengine/*" packages stopped being available.

Here are the ones we use:

* appengine (duh)
* appengine/datastore (replaced by Cloud Datastore)
* appengine/delay (Cloud Tasks)
* appengine/log (normal log?)
* appengine/memcache (no replacement, yay)
* appengine/urlfetch (just use net/http)

At least datastore and delay can be done before moving to the new runtime, I believe.

**Comment by bsiegert** [comment]
Hm, maybe I was mistaken, and the API migration can be postponed. I got something working with go111, which has the merit of not being deprecated.

**Comment by bsiegert** [comment]
Today's news: https://cloud.google.com/appengine/docs/standard/go/release-notes 

> Many legacy App Engine APIs are now available to select second-generation runtimes. These APIs are available for Go 1.12+ in preview, through language-idiomatic libraries. Calls to these APIs are billed according to the standard rates.

This goes in app.yaml:

```yaml
app_engine_apis: true
```

And in the code, for example:

```go
import (
  "google.golang.org/appengine/v2"
  "google.golang.org/appengine/v2/memcache"
)
```

**Comment by bsiegert** [comment]
I will focus on the standalone version with SQLite DB. Work on the App Engine version is stopped for now.
